net/snode.c/Makefile

```diff
@@ -0,0 +1,275 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=snode.c
+PKG_VERSION:=0.90
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/SNodeC/snode.c/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=49c3629f302e7760edf08430c1d6787c181a835b1db136c2dcc30a68186134c7
+
+# PKG_SOURCE_URL:=https://api.github.com/repos/SNodeC/snode.c/tarball/v$(PKG_VERSION)?
+# PKG_HASH:=446e470b5f33291e75078d775308bc0efdd184b07e93d6a4f18022bb5afd60b1
+
+PKG_MAINTAINER:=Volker Christian <me@vchrist.at>
+PKG_LICENSE:=LGPL-3.0-or-later
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_BUILD_DEPENDS:=nlohmannjson easyloggingpp
+PKG_BUILD_PARALLEL:=1
+
+CMAKE_INSTALL:=1
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+ifdef CONFIG_SNODEC_EPOLL
+  CONFIG_SNODEC_IO_MULTIPLEXER=epoll
+else ifdef CONFIG_SNODEC_POLL
+  CONFIG_SNODEC_IO_MULTIPLEXER=poll
+else ifdef CONFIG_SNODEC_SELECT
+  CONFIG_SNODEC_IO_MULTIPLEXER=select
+endif
+
+CMAKE_OPTIONS+=-DIO_MULTIPLEXER=$(CONFIG_SNODEC_IO_MULTIPLEXER) \
+               -DREAD_BLOCKSIZE=$(CONFIG_SNODEC_READ_BLOCKSIZE) \
+               -DWRITE_BLOCKSIZE=$(CONFIG_SNODEC_WRITE_BLOCKSIZE) \
+               -DREAD_TIMEOUT=$(CONFIG_SNODEC_READ_TIMEOUT) \
+               -DWRITE_TIMEOUT=$(CONFIG_SNODEC_WRITE_TIMEOUT) \
+               -DTERMINATE_TIMEOUT=$(CONFIG_SNODEC_TERMINATE_TIMEOUT) \
+               -DBACKLOG=$(CONFIG_SNODEC_BACKLOG) \
+               -DACCEPTS_PER_TICK=$(CONFIG_SNODEC_ACCEPTS_PER_TICK) \
+               -DREUSE_ADDRESS=$(CONFIG_SNODEC_REUSE_ADDRESS) \
+               -DREUSE_PORT=$(CONFIG_SNODEC_REUSE_PORT) \
+               -DIPV6_ONLY=$(CONFIG_SNODEC_IPV6_ONLY) \
+               -DIPV4_MAPPED=$(CONFIG_SNODEC_IPV4_MAPPED) \
+               -DACCEPT_TIMEOUT=$(CONFIG_SNODEC_ACCEPT_TIMEOUT) \
+               -DCONNECT_TIMEOUT=$(CONFIG_SNODEC_CONNECT_TIMEOUT) \
+               -DRECONNECT=$(CONFIG_SNODEC_RECONNECT) \
+               -DRECONNECT_TIME=$(CONFIG_SNODEC_RECONNECT_TIME) \
+               -DRETRY=$(CONFIG_SNODEC_RETRY) \
+               -DRETRY_TIMEOUT=$(CONFIG_SNODEC_RETRY_TIMEOUT) \
+               -DRETRY_TRIES=$(CONFIG_SNODEC_RETRY_TRIES) \
+               -DRETRY_BASE=$(CONFIG_SNODEC_RETRY_BASE) \
+               -DRETRY_JITTER=$(CONFIG_SNODEC_RETRY_JITTER) \
+               -DRETRY_LIMIT=$(CONFIG_SNODEC_RETRY_LIMIT) \
+               -DTLS_INIT_TIMEOUT=$(CONFIG_SNODEC_TLS_INIT_TIMEOUT) \
+               -DTLS_SHUTDOWN_TIMEOUT=$(CONFIG_SNODEC_TLS_SHUTDOWN_TIMEOUT) \
+               -DCMAKE_BUILD_WITH_INSTALL_RPATH=ON \
+               -DCMAKE_SKIP_RPATH=FALSE
+
+define Package/snode.c/Default
+  SECTION:=net
+  CATEGORY:=Network
+  SUBMENU:=SNode.C
+  URL:=https://github.com/VolkerChristian/snode.c
+  Maintainer:=Volker Christian <me@vchrist.at>
```

Comment:
This is already in PKG_MAINTAINER, drop.

Reply:
You are right - I have droped it.